packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -94,93 +97,97 @@ const Fieldset = ({
   };
 
   return (
-    <fieldset
-      data-h2-background="base(background)"
-      data-h2-display="base(flex)"
-      data-h2-align-items="base(flex-start)"
-      data-h2-gap="base(0, x.25)"
-      data-h2-border="base(none)"
-    >
-      <legend data-h2-visually-hidden="base(hidden)">{legend}</legend>
+    <Field.Fieldset flat>
+      <Field.Legend data-h2-visually-hidden="base(invisible)">
```

Comment:
No, that would create invalid markup. These are using `fieldset` elements that should have a `legend` as a direct child to provide a label for the group. If we use headings it opens a massive can of worms of using the correct level then assigning it to the group using `aria-*`

It is far more straight forward to just use a `legend` and follow HTML semantics. This way we provide a label for the group without leaning on aria which is always the preferred method.

Ref: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset

Would that wave concern mention anything about the font-weight? My guess would be that it is complaining that we are using bold for an entire line of text. This is a warning since users could confuse it with a heading but it is very subjective.

See: https://github.com/GCTC-NTGC/gc-digital-talent/issues/5004#issuecomment-1599152685

Reply:
The Wave alert is an alert, not an error.  I agree with your reason for going with the legend.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/548aca04-9d49-4a38-aa23-3944cc928bd5)

Reply:
Yeah, that is the message I was expecting. With the provided design it is hard to get around this 😢 So the markup it wants is:

```tsx
<fieldset>
 <legend />
  ...
</fieldset>
```

But because our "legend" is contained in a separate visual area as the controls (which should also be a part of the fieldset we need to do this:

```tsx
<fieldset>
  <legend visuall-hidden />
  <flex-container>
     <faux-fieldset>
       <p bold>Faux Legend</p> <-- it is complaining about this
     </faux-fieldset>
     <actions />
  </flex-container>
</fieldset>
```

Unfortunately, for some odd reason, `legend`s need to be direct children so we render a fake legend with bold text. We really _shouldn't_ be doing this but it is a limit of HTML that prevents us from implementing this layout properly without this alert 😢 

Currently, all I can think of is to move the legend outside of the `div` with the rounded edges and shadow but that could look silly.